PWGMM/Lumi/Tasks/CMakeLists.txt

```diff
@@ -10,10 +10,18 @@
 # or submit itself to any jurisdiction.
 
 o2physics_add_dpl_workflow(lumi
-                    SOURCES lumi.cxx
-                    PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
-                                          O2::ReconstructionDataFormats
-                                          O2::DetectorsBase
-                                          O2::DetectorsCommonDataFormats
-                                          O2::DetectorsVertexing
-                    COMPONENT_NAME Analysis)
+                             SOURCES lumi.cxx
```

Comment:
The indentation here has some extra whitespace, please remove it.

Reply:
Thank you. Done.